apps/web/src/pages/activities/services/chart-bar/option.service.ts

```diff
@@ -154,7 +154,7 @@ function updateToolTipStyles(context, tooltipEl: HTMLElement, tooltipModel) {
 
   /* eslint-disable no-param-reassign */
   tooltipEl.style.opacity = '1';
-  tooltipEl.style.left = `${position.left + window.scrollX + tooltipModel.caretX - tooltipModel.width - 40}px`;
+  tooltipEl.style.left = `${position.left + window.scrollX + tooltipModel.caretX - tooltipModel.width - 30}px`;
```

Comment:
I don't know why but this magic number worked before it was 40. 
I've tried to calculate it differently with `translateX` but it was not working.